### ISSUE TITLE
Trailing comma rule with mandatory comma should only trigger on multiline arrays

### ISSUE THREAD

**Issue by jkooker** [realm/SwiftLint#917]
The new trailing comma rule is very nice, and I want to turn it on for my multiline arrays. But that's triggering errors in my single-line arrays. It seems to me that the rule should only apply to multiline arrays.

In other words, can we make the following valid?

```swift
let x = [
    1,
    2,
    3,
]

let y = [1]
```

For .swiftlint.yml configuration:
```yaml
trailing_comma:
    mandatory_comma: true
```

**Comment by marcelofabri** [comment]
This is duplicate of https://github.com/realm/SwiftLint/issues/910 and was fixed in https://github.com/realm/SwiftLint/pull/912.

**Comment by jpsim** [comment]
I've cut a new release which includes this fix: [0.13.2: Light Cycle](https://github.com/realm/SwiftLint/releases/tag/0.13.2). Homebrew PR: https://github.com/Homebrew/homebrew-core/pull/7449
